.github/workflows/ci.yml

```diff
@@ -16,7 +16,11 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        laravel: [8, 9]
+        include:
+        - laravel: 9
+          php: 8.0
```

Comment:
Adding `php` here doesn't do anything right now I believe because we also need to use `matrix.php` in the actual job steps